spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceHiveTables.java

```diff
@@ -504,4 +504,75 @@ public synchronized void testHiveManifestsTable() throws Exception {
       });
     }
   }
+
+  @Test
+  public synchronized void testHivePartitionsTable() throws Exception {
```

Comment:
Can we also add a corresponding test to `TestIcebergSourceHadoopTables.java`?

Reply:
Yes, I thought I had. I'll update this.

Reply:
Done